CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## Not released
 - Add cypress e2e tests in CI [#194](https://github.com/CartoDB/carto-react-template/pull/194)
+- Remove case insensitive flag in source regex type check in layer generator [#198](https://github.com/CartoDB/carto-react-template/pull/198)
```

Comment:
@Clebal That's the solution, but what error does it fix? can you rephrase it a bit?

Reply:
Me ha quedado un poco largo, pero bueno, se entiende @VictorVelarde.